client/src/AppBundle/Controller/Admin/IndexController.php

```diff
@@ -225,14 +225,17 @@ public function editNdrAction(Request $request, $id)
      */
     public function deleteConfirmAction($id)
     {
+        /** @var EntityDir\User $userToDelete */
         $userToDelete = $this->getRestClient()->get("user/{$id}", 'User');
 
-        if (!$this->isGranted('ROLE_ADMIN')) {
-            throw new DisplayableException('Only Admin can delete users');
-        }
-
         /** @var EntityDir\User $loggedInUser */
         $loggedInUser = $this->getUser();
+
+        if (!$loggedInUser->isAdminOrSuperAdmin()) {
+            $message = $userToDelete->isAdminOrSuperAdmin() ? 'Only Super Admins can delete Admins or Super Admins' : 'Only Admins or Super Admins can delete users';
+            throw new DisplayableException($message);
```

Comment:
Should probably be (to throw a generic 403 response):
```suggestion
            throw new AccessDeniedHttpException($message);
```
or (to throw a custom error message):
```suggestion
            return $this->renderError($message, Response::HTTP_FORBIDDEN)
```

Reply:
Wasn't aware - I'll be on to trading standards about this error type.